yoti_python_sdk/dynamic_sharing_service/dynamic_scenario_builder.py

```diff
@@ -12,33 +12,31 @@ def __init__(self):
             "callback_endpoint": "",
         }
 
-    """
-    @param policy A DynamicPolicy defining the attributes to be shared
-    """
-
     def with_policy(self, policy):
+        """
+        :param policy: A DynamicPolicy defining the attributes to be shared
```

Comment:
I know this was here previously, but technically it's the attributes to be _requested_. Small difference, but with soft preferences, and potentially optional attributes in future, it's an important distinction 👍 
```suggestion
        :param policy: A DynamicPolicy defining the attributes to be requested
```